.github/scripts/check_config_changes_ci.py

```
@@ -0,0 +1,422 @@
+    api4_patches = {
+        fname: patch
+        for fname, patch in patches.items()
+        if fname.startswith("server/channels/api4/") and fname.endswith(".go")
+    }
+    if not api4_patches:
+        return result
+ 
+    added_eps:   list[str] = []
+    removed_eps: list[str] = []
+ 
+    for fname, patch in api4_patches.items():
```

Ruff (0.15.9)

[warning] Loop control variable `fname` not used within loop body

(B007)